docs: typos in AI overview (#74195)
metabase/metabase: docs/ai/overview.md

**Best for: asking ad-hoc, ephemeral questions; combining data from Metabase with data from other tools.**

> Connect your favorite third-party AI tool - like Claude or Codex - to the Metabase MCP server. MCP server's main functionality is designed for answering in-the-moment questions like "hey btw what's our q3 revenue?" The MCP server is also useful when combined with other MCP servers. For example, you can ask Claude a question about your customers that combines data from Metabase, your CRM, and your support ticket platform.

Reviewer: ```suggestion
Connect your favorite third-party AI tool - like Claude or Codex - to the Metabase MCP server. MCP servers are designed for answering in-the-moment questions like "hey btw what's our q3 revenue?" Metabase's MCP server is also useful when combined with other MCP servers. For example, you can ask Claude a question about your customers that combines data from Metabase, your CRM, and your support ticket platform.
```

> Compared to the built-in Metabot, MCP server has somewhat restricted functionality (for example, it can't generate code or build transforms for now). See [the ever-expanding list of MCP server tools](mcp.md#available-tools).

Reviewer: ```suggestion
Compared to the built-in Metabot, the MCP server has somewhat restricted functionality (for example, it can't generate code or build transforms for now). See [the ever-expanding list of MCP server tools](mcp.md#available-tools).
```

> **Controls**: Metabase MCP server requires people to authenticate into Metabase, and all the responses it provides will be scoped to their permissions. However, unlike built-in Metabot, MCP server doesn't come with granular control over which _tools_ people can use, or disable MCP server altogether.

Reviewer: ```suggestion
**Controls**: Metabase's MCP server requires people to log into Metabase, and all the responses it provides will be scoped to their permissions. However, unlike the built-in Metabot, our MCP server doesn't come with granular control over which _tools_ people can use, or the ability to disable MCP server altogether.
```

> **Provider**: Requests to Metabase MCP server are handled by the provider you choose to connect to the MCP server (e.g. Claude, Cursor, etc).

Reviewer: ```suggestion
**Provider**: Requests to Metabase's MCP server are handled by the provider you choose to connect to the MCP server (e.g. Claude, Cursor, etc).
```

**Best for: developers creating stuff that other people will use.**

> Use a coding agent like Claude Code to understand your database's metadata, generate Metabase content as YAML files locally, verify the schema, then sync and import the generated content into your production Metabase. Sky's the limit on what you can accomplish.

Reviewer: ```suggestion
Use a coding agent like Claude Code to understand your database's metadata, generate Metabase content as YAML files locally, verify the schema, then sync and import the generated content into your production Metabase. The sky's the limit on what you can accomplish.
```
